.github/workflows/run-tests.yml

```diff
@@ -74,9 +74,13 @@ jobs:
             cp sdk-specifications/features/access/authorization-failure-reporting.feature tests/acceptance/pam
             cp sdk-specifications/features/access/grant-token.feature tests/acceptance/pam
             cp sdk-specifications/features/access/revoke-token.feature tests/acceptance/pam
+            cp sdk-specifications/features/files/file-upload-to-space.feature tests/acceptance/files
+            cp sdk-specifications/features/history/history-vsp.feature tests/acceptance/history
 
             sudo pip3 install -r requirements-dev.txt
             behave --junit tests/acceptance/pam
+            behave --junit tests/acceptance/files
+            behave --junit tests/acceptance/history
```

Comment:
is it possible to do `behave --junit tests/acceptance/` and then be done with? 

Or maybe we could even forget about `cp` that we have above and do:

`behave --junit sdk-specifications/features --tags "something something"`. Check `–tags-help` for more. Here's how we use tags to run optional (those can fail and it's not blocking the build) and required in kotlin using tags:

* `not @skip and not @na=kotlin and @beta` those are optionals
* `not @skip and not @na=java and not @beta` those are required

Reply:
That way you probably have some step duplication, which means you could remove some code

Reply:
Currently behave doesn't have a feature to run tests in multiple subdirectories. Sadly we sometimes have couple of scenarios with the same name behaving slightly different. So for now it has to stay this way.